Messenger/AppDelegate.mm

```diff
@@ -105,6 +105,10 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
   _lastNotificationCount = @"";
 }
 
+- (void)setActiveConversationAtIndex:(NSString *)index {
+  [_webView.windowScriptObject evaluateWebScript:
+   [NSString stringWithFormat:@"document.querySelector('._1ht1:nth-of-type(%@) > a').click();", index]];
```

Comment:
There are. We're depending here on the order of elements in the DOM tree (taking the first matching element). But unless there are major changes to the Messenger layout, we should be fine.